features/support/cucumber_rails_helper.rb

```diff
@@ -33,27 +44,62 @@ def add_gem(name, *args)
     end
   end
 
-  private
+  def remove_gem(name)
+    content = File.read(expand_path('Gemfile')).gsub(/^\s*gem ["']#{name}["'].*$/, '')
+    overwrite_file('Gemfile', content)
+  end
+
+  def bundle_install
+    run_command_and_stop 'bundle config set --local without "development"'
+    run_command_and_stop "bundle config set --local path '#{ENV['GITHUB_WORKSPACE']}/vendor/bundle'" if ENV.key?('GITHUB_WORKSPACE')
+    run_command_and_stop 'bundle install --jobs 4'
+  end
+
+  def configure_rails_gems
+    %w[bootsnap byebug jbuilder listen rails sass-rails turbolinks webpacker].each { |gem| remove_gem(gem) }
+    %w[railties activerecord actionpack].each { |rails_gem| add_gem(rails_gem, Rails.version) }
+  end
+
+  def configure_rails_requires
+    content = File.read(expand_path('config/application.rb'))
+    %w[ active_job/railtie active_storage/engine action_mailer/railtie action_mailbox/engine
+        action_text/engine action_cable/engine rails/test_unit/railtie sprockets/railtie ].each do |require|
+      content = content.gsub(/^.*require ["']#{require}["']\s*$/, '')
+    end
+    overwrite_file('config/application.rb', content)
+  end
+
+  def configure_rails_layout
+    file = 'app/views/layouts/application.html.erb'
+    content = File.read(expand_path(file)).gsub(/^\s*<%= stylesheet_link_tag .*%>\s*$/, '')
+    overwrite_file(file, content)
```

Comment:
We've avoided compiling stylesheets via `sass-rails`, so remove the stylesheet link from the application layout.